cmd/brew-test-bot.rb

```diff
@@ -977,7 +980,10 @@ def test_ci_upload(tap)
 
     if (pr = ENV["UPSTREAM_PULL_REQUEST"])
       pull_pr = "https://github.com/#{tap.user}/homebrew-#{tap.repo}/pull/#{pr}"
-      safe_system "brew", "pull", "--clean", pull_pr
+      args = []
+      args << "--tap=#{@tap}" unless @tap.nil?
+      args << pull_pr
+      test "brew", "pull", "--clean", *args
```

Comment:
> My suggested solution would be to build bottles on Homebrew/homebrew-science if it's meant to support Linux

Can do. In fact I already have been doing for some time now. I originally built bottles on the Linuxbrew/science fork, but Linuxbrew/science can be decommissioned at this point. Close this issue?